src/content.rs

```diff
@@ -124,40 +105,19 @@ pub fn all_pages(
         }
         false => {
             let index_cache: IndexCache = all_pages_load(dir, show_unpublished)?;
-            let cache_expiry_time = match index_cache.cache_expiration {
-                Some(date) => date.to_rfc3339(),
-                _ => "0".to_string(),
-            };
+            let cache_expiry_time = index_cache.cache_expiration.to_rfc3339();
             // Serialize the files back out to disk for subsequent requests.
             let cache_data = serde_json::to_string(&index_cache.contents)?;
-
-            let cache_file = File::create(CACHE_FILE);
-            match cache_file {
-                Ok(f) => {
-                    let mut f = BufWriter::new(f);
-                    println!("Writing string  is {cache_expiry_time}");
-                    println!(
-                        "Writing Expiry Time is {:#?}",
-                        DateTime::parse_from_rfc3339(&cache_expiry_time)
-                    );
-                    // Cache_contents is stored in a file in the following manner
-                    // Cache expiry timer on the 1st line
-                    // The rest of the file contains the cache contents
-                    writeln!(f, "{cache_expiry_time}")?;
-                    write!(f, "{cache_data}")?;
-                }
-                _ => {
-                    eprintln!("Cannot create Cache file");
-                }
-            }
+            store.set("bartholomew_index_cache", cache_data)?;
```

Comment:
Comment on line 109 (re 'files') might be out of date?

Reply:
Yes - I will update the comment.